docs/index.rst

```diff
@@ -9,6 +9,7 @@ Client Types
 
    data_client/data_client_usage
    classic_client/usage
+   admin_client/admin_client_usage
```

Comment:
I probably would have preferred to have the new docs as a separate PR to keep things focused, but probably not a big deal